src/extension.ts

```diff
@@ -114,12 +116,13 @@ export async function activate(context: vscode.ExtensionContext) {
         )
 
         const api = {
-            version: 2,
+            version: 3,
             async getEnvironment() {
                 return await jlpkgenv.getAbsEnvPath()
             },
-            async getJuliaPath() {
-                return await juliaexepath.getJuliaExePath()
+            // TODO This is breaking, not sure how to handle that?
+            async getJuliaExecutable() {
+                return await g_juliaExecutablesFeature.getActiveJuliaExecutableAsync()

```

Comment:
@pfitzseb not sure this makes sense? I increased the version number, and then this will return a `JuliaExecutable` now...

Reply:
Yeah, this is fine. And I don't think anyone but me is using this API anyways :P